hivemq-edge/src/frontend/src/api/hooks/useFrontendServices/useGetConfiguration.tsx

```diff
@@ -6,8 +6,11 @@ import { ApiError, GatewayConfiguration } from '@/api/__generated__'
 export const useGetConfiguration = () => {
   const appClient = useHttpClient()
 
-  return useQuery<GatewayConfiguration, ApiError>([QUERY_KEYS.FRONTEND_CONFIGURATION], async () => {
-    const item = await appClient.frontend.getConfiguration()
-    return item
+  return useQuery<GatewayConfiguration, ApiError>({
+    queryKey: [QUERY_KEYS.FRONTEND_CONFIGURATION],
+    queryFn: async () => {
+      const item = await appClient.frontend.getConfiguration()
+      return item
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR

```suggestion
      return await appClient.frontend.getConfiguration()
```